curriculum/challenges/english/25-front-end-development/lecture-html-fundamentals/670803abcb3e980233da4768.md

```diff
@@ -0,0 +1,121 @@
+---
+id: 670803abcb3e980233da4768
+title: What Are Divs and Spans, and When Should You Use Them?
+challengeType: 11
+videoId: nVAaxZ34khk
+dashedName: what-are-divs-and-spans
+---
+
+# --description--
+
+Watch the video and answer the questions below.
+
+# --questions--
+
+## --text--
+
+What type of element is a `div` element?
```

Comment:
I feel like the wording here could be better clarified

maybe this?

```suggestion
What type of element is a `div`— inline, block, or inline-block?
```

Reply:
I think it works as is.